abdiff/core/utils.py

```diff
@@ -4,27 +4,25 @@
 import os
 from pathlib import Path
 
-from slugify import slugify
-
 from abdiff.config import Config
 
 CONFIG = Config()
 
 
-def get_job_slug_and_working_directory(job_name: str) -> tuple[str, Path]:
-    """Create working directory for new job by slugifying job name."""
-    job_slug = slugify(job_name)
-    return job_slug, Path(CONFIG.root_working_directory) / job_slug
+def read_job_json(job_directory: str) -> dict:
+    """Read job JSON file."""
+    job_json_filepath = Path(job_directory) / "job.json"
+    with open(job_json_filepath) as f:
+        return json.load(f)
 
 
-def update_or_create_job_json(job_name: str, new_job_data: dict) -> dict:
+def update_or_create_job_json(job_directory: str | Path, new_job_data: dict) -> dict:
```

Comment:
Do we still need the `Path` type hint if it is doing the `Path` conversion inside the function? Will later function pass the `job_directory` as a `Path`?

Reply:
Good call.  I had added it while working on the tests, where at one point it felt convenient to also pass a `Path` object... but that shifted and this remained.  Removed in upcoming commit.